packages/core/src/actions/smartAccount/internal/sendUserOperation.ts

```diff
@@ -31,14 +31,13 @@ export const _sendUserOperation: <
   }
 
   const request = deepHexlify(args.uoStruct);
-  if (!isValidRequest(request)) {
+  if (
+    !isValidRequest(request) ||
+    !/^0x[a-fA-F0-9]{130,}$/.test(request.signature)
```

Comment:
@moldy530 implemented this, any thoughts?

Also, strictly speaking, user ops could have signature formats other than this.

Reply:
tbh I can't remember why we added this. we can just remove it for now. the idea was to check that the signature is set and is at least 65bytes, but @adam-alchemy has pointed out that it can be smaller for some edge cases so we probs don't want this in core